### ISSUE TITLE
Test failure under Perl 5.24

### ISSUE THREAD

**Issue by cschwenz** [abeverley/Dancer2-Logger-Syslog#1]
The output of attempting to install `Dancer2::Logger::Syslog`:

```
cpan> install Dancer2::Logger::Syslog
Running install for module 'Dancer2::Logger::Syslog'
Checksum for /Users/cal/.cpan/sources/authors/id/A/AB/ABEVERLEY/Dancer2-Logger-Syslog-0.4.tar.gz ok
Configuring A/AB/ABEVERLEY/Dancer2-Logger-Syslog-0.4.tar.gz with Makefile.PL
Checking if your kit is complete...
Looks good
Generating a Unix-style Makefile
Writing Makefile for Dancer2::Logger::Syslog
Writing MYMETA.yml and MYMETA.json
  ABEVERLEY/Dancer2-Logger-Syslog-0.4.tar.gz
  /Users/cal/perl5/perlbrew/perls/perl-5.24.0/bin/perl Makefile.PL -- OK
Running make for A/AB/ABEVERLEY/Dancer2-Logger-Syslog-0.4.tar.gz
cp lib/Dancer2/Logger/Syslog.pm blib/lib/Dancer2/Logger/Syslog.pm
cp README.pod blib/lib/Dancer2/Logger/README.pod
Manifying 1 pod document
  ABEVERLEY/Dancer2-Logger-Syslog-0.4.tar.gz
  /usr/bin/make -- OK
Running make test
PERL_DL_NONLAZY=1 "/Users/cal/perl5/perlbrew/perls/perl-5.24.0/bin/perl" "-MExtUtils::Command::MM" "-MTest::Harness" "-e" "undef *Test::Harness::Switches; test_harness(0, 'blib/lib', 'blib/arch')" t/*.t
t/00-load.t ....... 1/1 # Testing Dancer2::Logger::Syslog 0.4, Perl 5.024000, /Users/cal/perl5/perlbrew/perls/perl-5.24.0/bin/perl
t/00-load.t ....... ok
t/01-log.t ........ 1/8
#   Failed test 'log works'
#   at t/01-log.t line 20.

#   Failed test 'log works with 'core' level'
#   at t/01-log.t line 21.

#   Failed test 'debug works'
#   at t/01-log.t line 23.

#   Failed test 'warning works'
#   at t/01-log.t line 24.

#   Failed test 'error works'
#   at t/01-log.t line 25.
# Looks like you failed 5 tests of 8.
t/01-log.t ........ Dubious, test returned 5 (wstat 1280, 0x500)
Failed 5/8 subtests
t/format.t ........ ok
t/pod-coverage.t .. ok
t/pod.t ........... ok

Test Summary Report
-------------------
t/01-log.t      (Wstat: 1280 Tests: 8 Failed: 5)
  Failed tests:  4-8
  Non-zero exit status: 5
Files=5, Tests=15,  1 wallclock secs ( 0.03 usr  0.02 sys +  0.95 cusr  0.11 csys =  1.11 CPU)
Result: FAIL
Failed 1/5 test programs. 5/15 subtests failed.
make: *** [test_dynamic] Error 255
  ABEVERLEY/Dancer2-Logger-Syslog-0.4.tar.gz
  /usr/bin/make test -- NOT OK
//hint// to see the cpan-testers results for installing this module, try:
  reports ABEVERLEY/Dancer2-Logger-Syslog-0.4.tar.gz
Failed during this command:
 ABEVERLEY/Dancer2-Logger-Syslog-0.4.tar.gz   : make_test NO

cpan>
```

The output of `perl -v`:

```
$ perl -v

This is perl 5, version 24, subversion 0 (v5.24.0) built for darwin-thread-multi-ld-2level
(with 1 registered patch, see perl -V for more detail)

Copyright 1987-2016, Larry Wall

Perl may be copied only under the terms of either the Artistic License or the
GNU General Public License, which may be found in the Perl 5 source kit.

Complete documentation for Perl, including FAQ lists, should be found on
this system using "man perl" or "perldoc perl".  If you have access to the
Internet, point your browser at http://www.perl.org/, the Perl Home Page.

$
```

**Comment by abeverley** [comment]
Thanks for the bug report. I'll look at this when I've got a moment. In the meantime (assuming you want to use the module) I recommend using `Dancer2::Logger::Logreport` instead, which does everything that this module does plus more.

**Comment by abeverley** [comment]
Sorry for mega-delay. Fixed with #2 - thanks @SysPete
